### DISCUSSION TITLE
Piio image optimisation, webp and avif?

### DISCUSSION THREAD

**Discussion by slanelb** [swellstores/community#160] [Help] [answered]
Hi, am I right in my understanding that Swell bought Piio?
Does this mean as users of Swell we get image optimisation and also webp and avif formats of our uploaded jpg images?

**Comment by ericingram** [comment, marked as answer]
Yes, that's right

**Reply by slanelb** [reply]
The very helpful Swell Help team confirmed that Swell automatically provides the best format version (avif, webp) from my uploaded jpegs. Swell will also automatically provide the best fallback options depending on browser support. 
Congratulations. I think you have a great product. Would love some documentation on how Swell manages images even if it's just to confirm how much automatic magic Swell provides (unless I missed it).

**Reply by ericingram** [reply]
Thanks for the feedback! We absolutely need to add new documentation for the Piio-powered CDN. Will make sure that gets in the pipeline.

**Comment by vimtor** [comment]
On this note it would be cool if the `File` resource for images included a `blurHash` for user blurred images as a fallback.
Similar to what Sanity.io does: https://www.sanity.io/docs/image-type#df3f768ce073

**Reply by ericingram** [reply]
Interesting. Will put it on our ideas backlog
